### PR TITLE
tighten scc test to start failing on more than six pods failing

### PR DESCRIPTION
**Pull request by deads2k**
This will help prevent regressions.

### CODE REVIEW

File: test/extended/util/client.go [hunk 1]
```diff
@@ -814,3 +817,37 @@ func defaultClientTransport(rt http.RoundTripper) http.RoundTripper {
...
+// installConfig The subset of openshift-install's InstallConfig we parse for this test
+type installConfig struct {
+	FIPS bool `json:"fips,omitempty"`
+}
+
+func IsFIPS(client clientcorev1.ConfigMapsGetter) (bool, error) {
+	// this currently uses an install config because it has a lower dependency threshold than going directly to the node.
+	installConfig, err := installConfigFromCluster(client)
+	if err != nil {
+		return false, nil
```
Comment (sttts) [inline test/extended/util/client.go:834]: not err?

File: test/extended/util/client.go [hunk 2]
```diff
@@ -814,3 +817,37 @@ func defaultClientTransport(rt http.RoundTripper) http.RoundTripper {
...
+}
+
+func installConfigFromCluster(client clientcorev1.ConfigMapsGetter) (*installConfig, error) {
+	cm, err := client.ConfigMaps("kube-system").Get(context.Background(), installConfigName, metav1.GetOptions{})
+	if err != nil {
+		return nil, err
+	}
+	data, ok := cm.Data["install-config"]
+	if !ok {
+		return nil, fmt.Errorf("No install-config found in kube-system/%s", installConfigName)
```
Comment (sttts) [inline test/extended/util/client.go:846]: nit: lowercase errors

**Comment by deads2k** [comment]
nits fixed.

**Pull request merged**